components/layout/PatternsLayout.tsx

```diff
@@ -0,0 +1,102 @@
+  Administration: "bg-administration",
+  Eligibility: "bg-eligibility",
+  "Security of tenure": "bg-security",
+  Develop: "bg-develop",
+  Stewardship: "bg-stewardship",
+  Use : "bg-use",
+} as const;
+
+const Header = () => (
+  <header className="bg-slate-200 p-8">
+    <h1 className="text-5xl mt-8 mb-8">Explore the patterns</h1>
+    <h2 className="text-2xl mb-2">What is ownership?</h2>
+    <p className="mb-8">Property is often described as a &quot;bundle&quot; of rights and obligations. We can unbundle these rights into 8 classes.</p>
+    <h2 className="text-2xl mb-2">What is a pattern?</h2>
+    <p>A pattern is a recurring solution. Although the precise details and the legal mechanisms used may vary in each case, by identifying these common patterns we can build a kind of &quot;genome&quot; for property; a library of building blocks that can be used to describe any form of ownership or tenure. Together, we are discovering new patterns all the time, as new entries are added to the Atlas.</p>
+  </header>
+);
+
+const PatternNav = (props: { patternClasses: PatternClass[] | undefined, onClick: Dispatch<SetStateAction<PatternClass | undefined>> }) => (
+  <nav className="columns-8 h-10 gap-0">
+    {props.patternClasses && props.patternClasses.map(patternClass => 
+      <button 
+        key={patternClass.name} 
+        className={`block w-full h-full text-left pl-2 text-sm ${patternClassLookup[patternClass.name]}`}
+        onClick={() => props.onClick(patternClass)}
+        >
+          {patternClass.name}
+        </button>
+    )}
+  </nav>
+);
+
+const PatternList = (props: { patternClass: PatternClass | undefined }) => {
+  const { patternClass } = props;
+  const { data: patternInfo, error: patternInfoError } = trpc.patternInfo.useQuery({ patternClassName: patternClass?.name || null } )
+
+
+ return (
+  <section className="p-8">
+    <p className="mb-4">{patternClass?.description}</p>
+    <h3 className="text-lg mb-4">Rights</h3>
+     {patternInfo?.rights.map(pattern => (
+       <PatternItem key={pattern.name} pattern={pattern} patternClassName={patternClass?.name} highestCount={patternInfo?.rights[0].entryCount} />
+     ))}
+    <h3 className="text-lg mb-4">Obligations</h3>
+     {patternInfo?.obligations.map(pattern => (
+       <PatternItem key={pattern.name} pattern={pattern} patternClassName={patternClass?.name} reverse highestCount={patternInfo?.obligations[0].entryCount} />
+     ))}
+  </section>
+ )
+}
+
+const PatternItem = (props: { pattern: Pattern, patternClassName: string | undefined, reverse?: boolean, highestCount: number | undefined }) => {
+  const { pattern, patternClassName, reverse, highestCount } = props
+
+  return (
+    <div className={`flex mb-4 ${reverse ? "flex-row-reverse" : ""}`}>
+      <div className={clsx(`w-1/2 ${patternClassLookup[patternClassName!]} bg-opacity-40 flex items-center justify-center`)}>
+        <LogoGithub size={32} className="w-1/4 flex-center"/>       
```

Comment:
i hadn't really registered from the figma designs that these patterns were going to link to github - are they on github already (in a repo? as a gist?)? that might be a notion task ticket we're missing!

Reply:
Sorry! This is me being a bit too quick/dirty maybe 😅 

It's just a placeholder logo for the pattern logos which are to come - I just copy/pasted an icon from the footer here.

Reply:
ha no worries! open cirlcle or simple geometric shape would be a nice swap if easy, but no worries if not! octocat just has such specific meanings to me :octocat: